salesagility/SuiteCRM
# Update Jquery js libraries

## Description
Update Jquery js libraries to current support versions on

- JQuery-UI js and CSS
- jstree plugin 

## How To Test This
See everything UI wise still works and displays as expected
jstree specifically used in email List view

## Types of changes


### Final checklist
